Source/WebKit/Configurations/BaseXPCService.xcconfig

```diff
@@ -37,6 +37,7 @@ APP_ICON_Release = WebKit.icns;
 APP_ICON_Production = ;
 
 EXCLUDED_SOURCE_FILE_NAMES[sdk=embedded*] = $(EXCLUDED_SOURCE_FILE_NAMES_$(CONFIGURATION)) *.xib MediaFormatReader.bundle;
+EXCLUDED_SOURCE_FILE_NAMES[sdk=iphoneos17.4*] = *;
```

Comment:
These will need to change for future iPhone OS SDKs.

Reply:
Yeah, I would recommend inverting it. Set `EXCLUDED_SOURCE_FILE_NAMES = *` and then set `INCLUDED_SOURCE_FILE_NAMES[sdk=iphone*x.y] = *` for iOS 17.0 - 17.3.

Reply:
Ah, good point! Will fix.